### ISSUE TITLE
Incorrect CURL Syntax on "Design Documents" page

### ISSUE THREAD

**Issue by JohnMcW** [oreilly/couchdb-guide#427]
curl -X POST http://127.0.0.1:5984/basic -d '{}'

Gives error:

{"error":"bad_content_type","reason":"Content-Type must be application/json"}

Should be:

curl -X POST http://127.0.0.1:5984/basic -d '{}' -H"Content-Type: application/json"

EDIT - using CouchDB v0.11.0 on Debian Squeeze

**Comment by ghost** [comment]
Agree. I also have CouchDB on Debian Squeeze, and I'm getting the very same error.
